### DISCUSSION TITLE
Multiple passes height setting

### DISCUSSION THREAD

**Discussion by DonDepresor** [arkypita/LaserGRBL#1563] [Q&A] [answered]
Hi y'all.
I have noticed that when I select multiple passes the height of the z-axis varies from pass to pass. How much does the z-axis drop between two consecutive passes? How can this value be adjusted?
Thanks in advance.

**Comment by gmmanonymus111** [comment, marked as answer]
You can edit that value in `Settings`->`GCode`->`Multiple passes`

**Reply by DonDepresor** [reply]
Thanks!!
I thought that i knew how to read, but may be i must return to school... I've looking over the settings menu for a long time... may be i must buy new spectacles.
